packages/clerk-js/src/ui/components/OrganizationSwitcher/OrganizationSwitcherPopover.tsx

```diff
@@ -85,7 +86,7 @@ export const OrganizationSwitcherPopover = React.forwardRef<HTMLDivElement, Orga
       if (createOrganizationMode === 'navigation') {
         return navigateCreateOrganization();
       }
-      return openCreateOrganization({ afterCreateOrganizationUrl, skipInvitationScreen });
+      return openCreateOrganization({ afterCreateOrganizationUrl, skipInvitationScreen, hideSlug });
```

Comment:
What y'all think about putting these 3 properties in a `createOrganizationProps` instead and add deprecation notice to `afterCreateOrganizationUrl` and `skipInvitationScreen`?

This will align with the `organizationProfileProps` prop